tests/operators/test_python_operator.py

```diff
@@ -85,6 +87,10 @@ def setUp(self):
         self.addCleanup(self.dag.clear)
         self.clear_run()
         self.addCleanup(self.clear_run)
+        try:
```

Comment:
Minor point, but since we are making other changes: could you move this to tearDown as it is cleanup code

Reply:
done